source/releasenotes/2024-11-22-php-81-82-83.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "PHP 8.1, 8.2 and 8.3 updated to their latest patch releases"
```

Comment:
```suggestion
title: "PHP 8.1, 8.2 and 8.3 updated to their latest security releases"
```

Maybe worth calling out that this is a security release?

Reply:
The 8.3/8.2 releases also include bug fixes. I think patch is fine.